rocAL/rocAL/include/video_loader.h

```diff
@@ -57,6 +57,8 @@ class VideoLoader : public VideoLoaderModule
     void set_prefetch_queue_depth(size_t prefetch_queue_depth) override;
     std::vector<size_t> get_sequence_start_frame_number();
     std::vector<std::vector<float>> get_sequence_frame_timestamps();
+    void shut_down() override;
```

Comment:
Looks like, you haven't merged the tot. Many changes which is already there in tot is showing here